crates/lair_keystore/src/bin/lair-keystore-bin/main.rs

```diff
@@ -223,8 +224,6 @@ async fn exec() -> LairResult<()> {
 }
 
 #[tokio::main(flavor = "multi_thread")]
-async fn main() {
-    if let Err(e) = exec().await {
-        eprintln!("{e}");
-    }
+async fn main() -> LairResult<()> {
+    exec().await
 }
```

Comment:
:+1: - This is a good update. And though it's technically a breaking change, it feels like a change that is making the behavior meet expectations, so it might even be acceptable to include it without a bigger version bump. But we can decide that separately.